docker/Dockerfile-ubuntu-20.04

```diff
@@ -1,44 +1,11 @@
 FROM ubuntu:focal
 ENV DEBIAN_FRONTEND noninteractive
 COPY build-and-install-scafacos.sh /tmp/
-RUN apt-get update && apt-get install -y \
-    apt-utils \
-    cmake \
-    ccache \
-    autoconf \
-    gfortran \
-    build-essential \
-    pkg-config \
-    openmpi-bin \
-    libopenmpi-dev \
-    clang-9 \
-    clang-tidy-9 \
-    clang-format-9 \
```

Comment:
why remove clang? this image is no longer going to replace `clang:6.0`?

Reply:
will fix